src/Backend/Modules/Pages/Installer/Installer.php

```diff
@@ -107,7 +107,7 @@ private function installExampleData(): void
                     ['extra_id' => $extras['blog_widget_categories'], 'position' => 'main'],
                     ['extra_id' => $extras['blog_widget_archive'], 'position' => 'main'],
                     ['extra_id' => $extras['blog_widget_recent_articles_list'], 'position' => 'main'],
-                    ['extra_id' => $extras['search_form'], 'position' => 'top']
+                    ['extra_id' => $this->getExtraId('sitemap'), 'position' => 'top']
```

Comment:
This shouldn't be sitemap but it should be the search_form, it is a bug that has slipped in when the installer was updated